src/CommunityToolkit/Xamarin.CommunityToolkit/Views/ViewSwitcher/TextSwitcherRenderer.android.cs

```diff
@@ -0,0 +1,558 @@
+			{
+				return;
+			}
+
+			var realWidth = (int)Context.ToPixels(width);
+			var realHeight = (int)Context.ToPixels(height);
+
+			var widthMeasureSpec = MeasureSpecFactory.MakeMeasureSpec(realWidth, MeasureSpecMode.Exactly);
+			var heightMeasureSpec = MeasureSpecFactory.MakeMeasureSpec(realHeight, MeasureSpecMode.Exactly);
+
+			Measure(widthMeasureSpec, heightMeasureSpec);
+		}
+
+		protected override void Dispose(bool disposing)
+		{
+			if (disposed)
+				return;
+
+			disposed = true;
+
+			if (disposing)
+			{
+				if (Element != null)
+				{
+					Element.PropertyChanged -= OnElementPropertyChanged;
+				}
+
+				BackgroundManager.Dispose(this);
+				if (visualElementTracker != null)
+				{
+					visualElementTracker.Dispose();
+					visualElementTracker = null;
+				}
+
+				if (visualElementRenderer != null)
+				{
+					visualElementRenderer.Dispose();
+					visualElementRenderer = null;
+				}
+
+				children.ForEach(c => c.Dispose());
+
+				spannableString?.Dispose();
+				Platform.ClearRenderer(this);
+			}
+
+			base.Dispose(disposing);
+		}
+
+		public override bool OnTouchEvent(MotionEvent? e)
+		{
+			if (visualElementRenderer.OnTouchEvent(e) || base.OnTouchEvent(e))
+			{
+				return true;
+			}
+
+			return motionEventHelper.HandleMotionEvent(Parent, e);
+		}
+
+		protected virtual void OnElementChanged(ElementChangedEventArgs<Label> e)
+		{
+			ElementChanged?.Invoke(this, new VisualElementChangedEventArgs(e.OldElement, e.NewElement));
+
+			if (e.OldElement != null)
+			{
+				e.OldElement.PropertyChanged -= OnElementPropertyChanged;
+				this.MaybeRequestLayout();
+			}
+
+			if (e.NewElement != null)
+			{
+				this.EnsureId();
+
+				if (visualElementTracker == null)
+				{
+					visualElementTracker = new VisualElementTracker(this);
+				}
+
+				e.NewElement.PropertyChanged += OnElementPropertyChanged;
+
+				UpdateText();
+				UpdateLineHeight();
+				UpdateCharacterSpacing();
+				UpdateTextDecorations();
+				UpdateTransition();
+				if (e.OldElement?.LineBreakMode != e.NewElement.LineBreakMode)
+					UpdateLineBreakMode();
+				if (e.OldElement?.HorizontalTextAlignment != e.NewElement.HorizontalTextAlignment || e.OldElement?.VerticalTextAlignment != e.NewElement.VerticalTextAlignment)
+					UpdateGravity();
+				if (e.OldElement?.MaxLines != e.NewElement.MaxLines)
+					UpdateMaxLines();
+
+				UpdatePadding();
+
+				ElevationHelper.SetElevation(this, e.NewElement);
+			}
+		}
+
+		protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+		{
+			if (this.IsDisposed())
+			{
+				return;
+			}
+
+			ElementPropertyChanged?.Invoke(this, e);
+
+			if (Control?.LayoutParameters == null && hasLayoutOccurred)
+				return;
+
+			if (e.PropertyName == Label.HorizontalTextAlignmentProperty.PropertyName || e.PropertyName == Label.VerticalTextAlignmentProperty.PropertyName)
+				UpdateGravity();
+			else if (e.PropertyName == Label.TextColorProperty.PropertyName ||
```

Comment:
Do we have the help method `e.PropertyName.Is()` is used on Forms to make those checks shorter and easy to read, I think that we could bring it from here

Reply:
Let's move it outside of this PR please